frontend/src/components/DocCardBadges.tsx

```diff
@@ -35,7 +35,7 @@ const DocCardBadges: React.FC<DocCardBadgesProps> = ({ odhDoc }) => {
   const label = DOC_TYPE_LABEL[docType] || 'Documentation';
 
   return (
-    <div className="odh-card__doc-badges">
+    <LabelGroup defaultIsOpen numLabels={5}>
```

Comment:
Why 5? It looks like you best have 3?

Reply:
Oh sorry, this is also for testing only, forgot to clean up 🤦‍♂️ but yeah, we will only have 2-3 labels at maximum